File: backend/src/main/java/project/backend/domain/chat/chatmessage/dto/event/EventMessageResponse.java
package project.backend.domain.chat.chatmessage.dto.event;

import java.time.LocalDateTime;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import project.backend.domain.chat.chatmessage.entity.MessageType;

@Getter
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class EventMessageResponse {

	private Long messageId;

	private MessageType type;

	private String sender;

	private Long roomId;

	private String content;

	private LocalDateTime sendAt;

}

File: backend/src/main/java/project/backend/domain/chat/chatmessage/dto/event/EventMessageResponse.kt
package project.backend.domain.chat.chatmessage.dto.event

import project.backend.domain.chat.chatmessage.entity.MessageType
import java.time.LocalDateTime

class EventMessageResponse (
    val messageId: Long?,

    val type: MessageType,

    val sender: String,

    val roomId: Long,

    val content: String,

    val sendAt: LocalDateTime = LocalDateTime.now()
)

